compiler/src/iree/compiler/Preprocessing/Passes.cpp

```diff
@@ -82,6 +82,8 @@ void buildPreprocessingPassPipeline(
         preprocessingOptions.preprocessingTransformSpecFilename;
     passManager.addPass(
         Preprocessing::createInterpreterPass(interpreterOptions));
+    // Custom dispatch formation creates util.func ops that need to be inlined.
+    passManager.addPass(mlir::createInlinerPass());
```

Comment:
```suggestion
```
run this pass from your script instead - "custom dispatch formation" is just one thing the interpreter is used for and not something we want to reference/bake in here

(we should fix the bugs that require the inliner to be run, but for today the workaround is to run it yourself)